crates/bindings-csharp/BSATN.Runtime/Db.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SpacetimeDB;
+
+public abstract class DbContext<DbView>
```

Comment:
Let's turn this into `abstract record`. Looks like in all the descendants it's just a data holder for things passed to the constructor, and `record`s have nicer syntax for this use-case.